linkerd/app/inbound/src/http/tests.rs

```diff
@@ -116,11 +96,11 @@ async fn downgrade_origin_form() {
     // Build the outbound server
     let cfg = default_config();
     let (rt, _shutdown) = runtime();
-    let server = build_server(cfg, rt, profiles, connect).new_service(accept);
+    let server = build_server(cfg, rt, profiles, connect).new_service(Target::H2);
     let (mut client, bg) = http_util::connect_and_accept(&mut client, server).await;
 
     let req = Request::builder()
-        .method(http::Method::GET)
+        .method(::http::Method::GET)
         .uri("/")
         .header(http::header::HOST, "foo.svc.cluster.local")
```

Comment:
why is the leading `::` necessary for the method but not the header here?

Reply:
`http` isn't the http crate, it's the `proxy::http` module. I've re-exported `Method` so the difference isn't needed.